### ISSUE TITLE
For days Markdown for links has not worked.

### ISSUE THREAD

**Issue by Rogue-Git-Dev** [vector-im/element-web#25398]
### Steps to reproduce

I am using nightly builds on Mac OS version 13.4 latest public release candidate. It is simple. I use Markdown for links in the format of [link name](url). For a few days [not sure when it began] Element fails to render links nicely as did previously. Instead, the code shows and the URL shows. No settings were changed and Markdown was verified to be enabled.

#### What did you expect?
Links are expected to be rendered nicely.
#### What happened instead?
Links were rendered as URLs.

### Operating system

Mac OS

### Application version

Element Nightly version: 2023051701 Olm version: 3.2.14

### How did you install the app?

from the nightly page on the Element website

### Homeserver

matrix.org default server

### Will you send logs?

Yes

**Comment by Rogue-Git-Dev** [comment]
Having been writ by Michael Telatynski on Thursday 18 May 2023 at 
04:00:58 -0500 ⬇️

> Looks fine for me on develop.element.io
I just updated Element on Mac to the latest nightly build, sent someone 
a message with a link in the name of [this text cloaks the URL](url). 
The actual text does not display for me but only the URL. This is not 
expected behaviour.

-- 
Brother J.

**Comment by t3chguy** [comment]
@Rogue-Git-Dev please provide the `... -> View Source` for the event in question
What client sent the event? Can you provide repro steps? A video?

**Comment by Rogue-Git-Dev** [comment]
Having been writ by Michael Telatynski on Friday 19 May 2023 at 05:34:42 
-0500 ⬇️

> @Rogue-Git-Dev please provide the `... -> View Source` for the event 
> in question
 From the More Options | View Source dialogue:
--------------------------------
Decrypted Event Source:
{
   "type": "m.room.message",
   "content": {
     "msgtype": "m.text",
     "body": "I mentioned [this forthcoming 
capability](https://9to5mac.com/2023/05/19/everyone-should-use-personal-voice/) 
is unnecessary. They in this article make the illogical case that 
sounding robotic is in some way bad or negative. I know and I am sure 
you know that is unequivocally false. If I were to lose my voice then 
this is the last thing I would do. There is something comforting about 
thinking robotically, sounding, and being robotic whether or not one has 
got their own voice.",
     "format": "org.matrix.custom.html",
     "formatted_body": "I mentioned <a 
href=\"https://9to5mac.com/2023/05/19/everyone-should-use-personal-voice/\">this 
forthcoming capability</a> is unnecessary. They in this article make the 
illogical case that sounding robotic is in some way bad or negative. I 
know and I am sure you know that is unequivocally false. If I were to 
lose my voice then this is the last thing I would do. There is something 
comforting about thinking robotically, sounding, and being robotic 
whether or not one has got their own voice."
   }
}
Original Event Source:
Redacted unless provision is necessary
> What client sent the event?
Mac OS version 13.4 build 22F66

-- 
Brother J.

**Comment by t3chguy** [comment]
I copy and pasted that event content into `/devtools` and it looks fine to me 

![image](https://github.com/vector-im/element-web/assets/2403652/7a6f6379-0299-4139-b2a1-b126cfb1a562)

**Comment by Rogue-Git-Dev** [comment]
Having been writ by Michael Telatynski on Friday 19 May 2023 at 07:29:31 -0500 ⬇️

> I copy and pasted that event content into `/devtools` and it looks fine to me
Why for the past four days approximately has it not displayed as did previously to me then? It displays as URL without the text to cloak it. It is quite strange.

-- 
Brother J.

**Comment by t3chguy** [comment]
No clue, you haven't shared repro steps, a screen recording or even a screenshot.

**Comment by t3chguy** [comment]
Regressed by https://github.com/matrix-org/matrix-react-sdk/commit/99ac9e502987da8927b131f13d586445cf046831 - its due to Element Desktop showing a tooltip to the actual URL to prevent link spoofing and that being read instead of the content, not intended

**Comment by t3chguy** [comment]
That change was added 2 weeks ago, working on a fix to workaround it now.

**Comment by t3chguy** [comment]
The solution is to set the url as the description rather than the label

**Comment by Rogue-Git-Dev** [comment]
Having been writ by Michael Telatynski on Friday 19 May 2023 at 08:17:18 -0500 ⬇️

> The solution is to set the url as the description rather than the label
Very well then. I anticipate to test it. It will be nice to hear the underlying destination provided it works as I envisage it.

-- 
Brother J.
